README.md

```diff
@@ -99,60 +99,60 @@ You can get it from http://repo1.maven.org/maven2/org/glassfish/javax.json/1.0.4
+When the javax.json-1.0.4.jar, commons-codec-1.1.jar, and maximo-restclient-0.1.jar files are ready, add them to the Java project as common reference libraries.
 	
 # III. Usage
 -----
 
-Maximo Resources (Object Structures) represent a graph of related Maximo objects (Mbo's) that provides an atomic view/object to create/update/delete/query the releated set of Mbos. 
+Maximo Resources, or object structures, represent a graph of related Maximo business objects (Mbos) that provides an atomic view/object to create, update, delete, and query the releated set of Mbos. 
 
-We will use Work Order, Purchase Order and Companies Resource as examples to show you how to use the Maximo Rest Client.
+We will use the Work Order, Purchase Order and Companies Resources as examples to show you how to use the Maximo REST Client.
 
->**note**: The use cases can be find at TestOSLCApi.java
+>**Note**: The use cases can be found at TestOSLCApi.java.
 
-## 3.1 Query a Work Order  for Work Order Set (mxwodetail)
+## 3.1 Query a work order for work order set (mxwodetail)
 
-The following instruction shows how to query a work order from Maximo by using Maximo Rest Client Library.
+The following instruction shows how to query a work order from Maximo Asset Management by using the Maximo RET Client Library.
 
-### 3.1.1 Connect to Maximo
+### 3.1.1 Connect to Maximo Asset Management
 
-Before we connect to Maximo, it is necessary to set up authentication and environment information in Options;
+Before you connect, it is necessary to set up the authentication and environment information in Options.
 
-* For authentication, it needs username, password and authentication way among "maxauth", "basic" and "form". The sample code is as following,
+* For authentication, the username, password and authentication method are required. The value for authentication method can be "maxauth", "basic" or "form". The sample code is shown in the following code:
 
 ```java
 Options option = new Options().user("maxadmin").password("maxadmin").auth("maxauth");
 ```
 
-> **note**: For Maximo Multi-Tenancy, take the tenant code = "00" as an example, using the following Options.
+> **Note**: For Maximo Asset Management Multitenancy, take the tenant code = "00", as an example, in the following Options.
 
 ```java
 Options option = new Options().user("maxadmin").password("maxadmin").auth("maxauth").mt(true).tenantCode("00");
 ```
 
-* For environment, it needs the data mode setting, host, port and if it the debug is enabled. The sample code is as following,
+* For environment, it needs the data mode setting, host, port, and if  the debug is enabled. The sample code is shown in the following code:
 
 ```java
 option.host("host").port(7001).lean(true);
 ```
 
-* Based on this configuration, connect to the Maximo using MaximoConnector.
+* Based on this configuration, connect to the Maximo Asset Management by using MaximoConnector:
```

Comment:
line 119, 139, 195
 For clarity, it would help to have an article and identifying noun here. 

the MaximoConnector method?